README.md

```diff
@@ -180,10 +180,10 @@ Then, run the dev loop:
 1. Make your code changes
 1. Rebuild and re-run the package by running the following from the root of the repo:
    ```bash
-   kurtosis run . --args "{}"
```

Comment:
Isn't it equivalent to `kurtosis run .`?

Reply:
yes i think kevin wanted to highlight it can be changed when he wrote the preivous readme.